src/notte/common/notifier/mail.py

```diff
@@ -0,0 +1,85 @@
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+from pydantic import BaseModel
+from typing_extensions import override
+
+from notte.common.agent.types import AgentResponse
+from notte.common.notifier.base import BaseNotifier
+
+
+class EmailConfig(BaseModel):
```

Comment:
can you add one by default for the Notte Email server ? ask @andreakiro for the credentials. The `sender_password` should be taken from the .env file.